services/scabbard/libscabbard/Cargo.toml

```diff
@@ -86,13 +86,13 @@ authorization = ["splinter/authorization"]
 client = []
 client-reqwest = ["client", "reqwest"]
 commit-store = []
-diesel-receipt-store = ["diesel"]
+diesel-receipt-store = ["diesel", "sawtooth/transaction-receipt-store-lmdb"]
 events = ["splinter/events"]
 factory-builder = []
 https = []
 lmdb = []
-postgres = ["diesel/postgres", "diesel_migrations"]
+postgres = ["diesel/postgres", "diesel_migrations", "sawtooth/diesel", "sawtooth/postgres"]
```

Comment:
We should be able to simplify this such that we can add "sawtooth/postgres" but not "sawtooth/diesel" since "sawtooth/postgres" should pull in diesel.

Reply:
Yeah, removed.